### PR TITLE
feat(features/home): update profile name

### PR DESCRIPTION
**Pull request by olgahaha**
### Changed

1. Display and update profile name instead of username
1. Reduce duplicate API calls for profile

### Removed

1. Remove unused pages: `ProfilePage` and `PhoneVerificationPage`, and the service `DiaBackendAssetRefreshingService`

​
┆The issue is also created on [Asana](https://app.asana.com/0/1201016280880500/1208625719723109)

### CODE REVIEW

File: src/app/features/home/edit-profile/edit-profile.page.ts [hunk 1]
```diff
@@ -27,22 +19,21 @@ import { NetworkService } from '../../../shared/network/network.service';
...
+  readonly avatar$ = this.diaBackendAuthService.profile$.pipe(
+    map(profile => profile.profile_picture_thumbnail)
+  );
+  readonly background$ = this.diaBackendAuthService.profile$.pipe(
     map(profile => profile.profile_background_thumbnail)
   );
   readonly form = new UntypedFormGroup({});
   model: EditProfileFormModel = {
-    username: '',
+    displayName: '',
```
Comment (shc261392) [inline src/app/features/home/edit-profile/edit-profile.page.ts:36]: Found it in some cases it referred to as profileName (e.g. transaltion) and sometimes displayName. Maybe use profileName for all Ionic side variables and use `display_name` only on backend API response?

File: src/app/features/profile/phone-verification/phone-verification.page.html
Comment (shc261392) [inline src/app/features/profile/phone-verification/phone-verification.page.html:1]: Just confirming, Is phone verification intended to be remove in the change?

Comment (olgahaha) [reply src/app/features/profile/phone-verification/phone-verification.page.html:1]: I checked the code, and I didn’t find any references to it. I don’t think we need to update the code if there are no references that use the username. If I missed something, please correct me.

Comment (shc261392) [reply src/app/features/profile/phone-verification/phone-verification.page.html:1]: Seems it's already removed prior to this change.

**Review by shc261392 — APPROVED** [review]

**Pull request merged**